client/src/constants/validation.js

```diff
@@ -213,6 +213,14 @@ export const HiredParticipantSchema = yup.object().noUnknown('Unknown field in f
   site: yup.number().required('Site is required'),
 });
 
+export const EditParticipantFormSchema = yup.object().shape({
+  firstName: yup.string().required(errorMessage),
+  lastName: yup.string().required(errorMessage),
+  phoneNumber: yup.string().required(errorMessage).matches(/^\d{10}$/, 'Phone number must be provided as 10 digits'),
+  emailAddress: yup.string().required(errorMessage).matches(/^(.+@.+\..+)?$/, 'Invalid email address'),
+  //postalCode: yup.string().required(errorMessage).matches(/^[A-Z]\d[A-Z]\s?\d[A-Z]\d$/, 'Format as A1A 1A1'),
```

Comment:
Commented code can be removed :)

Do we want validation on the interest dropdown pick as well?

Reply:
I'm not sure. For the users that don't have an interest value, do we want selecting an option to be mandatory?

Reply:
Good point, I don't think we want it mandatory in that case. Can we make it nullable and ensure it's an expected value if entered though?